src/main/resources/templates/fragments/selected-grouping.html

```diff
@@ -10,41 +10,43 @@
                 ng-click="returnToManagePerson(); cancelDescriptionEdit(); toggleShowAdminTab();">
             <i class="fas fa-arrow-left mr-1"></i> Return to Manage Person
         </button>
+
         <!-- Export to CSV button -->
-        <div class="btn-group" id="csvButton">
-            <div class="d-flex" tooltip data-original-title="{{ getCSVToolTipMessage() }}">
-                <button type="button" class="btn btn-primary btn-dropdown-split"
-                        ng-disabled="!paginatingComplete || groupingMembers.length === 0"
-                        ng-click="exportGroupToCsv(groupingMembers, 'members')">
-                    <i class="fa fa-users mr-1"></i>
-                    Export Members
-                </button>
-            </div>
-            <button type="button" class="btn btn-primary dropdown-toggle-split dropdown-toggle" data-toggle="dropdown"
-                    aria-label="More export options" aria-haspopup="true" aria-expanded="false"
-                    ng-disabled="!paginatingComplete || (groupingMembers.length === 0 && groupingExclude.length === 0)">
+        <div class="btn-group dropdown" id="csvButton" tooltip data-original-title="{{ getCSVToolTipMessage() }}">
+            <button type="button" class="form-control" data-toggle="dropdown"
+                    aria-haspopup="true" aria-expanded="false"
+                    ng-disabled="!paginatingComplete || (groupingMembers.length === 0 && groupingBasis.length === 0 && groupingInclude.length === 0 && groupingExclude.length === 0)">
```

Comment:
Could you put this long condition: `(groupingMembers.length === 0 && groupingBasis.length === 0 && groupingInclude.length === 0 && groupingExclude.length === 0)` into its own function? I think isGroupingEmpty() would be a good name. Also add a Jasmine test for it. Thanks!